src/features/game/events/landExpansion/garbageSold.test.ts

```diff
@@ -168,4 +168,12 @@ describe("garbageSold", () => {
       amount,
     );
   });
+
```

Comment:
When using `useFakeTimers` in jest its important to always reset the timers after using `useRealTimers`. You can read about it in the [docs](https://jestjs.io/docs/next/jest-object#fake-timers) 

Basically jest will continue to mock the timers until they're reset. You can either do it in a `beforeEach` hook or at the end of the test 🤗